db/migrate/20180809155959_add_conversion_host_id_to_miq_tasks.rb

```diff
@@ -0,0 +1,6 @@
+class AddConversionHostIdToMiqTasks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_tasks, :conversion_host_id, :bigint
```

Comment:
Is this necessary?  I'm assuming that the `Host` would be the resource on the `ConversionHost`, right?

Something like:
```ruby
class ConversionHost < ApplicationRecord
  belongs_to :resource, :polymorphic => true
  ...
end

class Host < ApplicationRecord
  has_one :conversion_host, :as => :resource
  ...
end
```